### ISSUE TITLE
rule: `empty-file`

### ISSUE THREAD

**Issue by felipeasimos** [DonIsaac/zlint#244]
## Description
This rule would check for empty `.zig` files in the project. An empty `.zig` is probably left by accident, it makes sense to flag it in the linter. A file should be deemed empty if it has no content (zero bytes) or only whitespace characters

## Examples

Examples of **incorrect** code for this rule (empty file)
```zig
```

Examples of **correct** code for this rule
```zig
// non-empty file
fn main() void {
}
```
